molecule/user_provided/prepare.yml

```diff
@@ -0,0 +1,81 @@
+---
+- name: Prepare
+  hosts: localhost
+  connection: local
+  gather_facts: false
+  become: yes
+  tasks:
+    - name: stopping service
+      ansible.builtin.service:
+        name: mariadb
+        state: stopped
+
+    - name: uninstalling
+      ansible.builtin.yum:
+        name:
+          - mariadb
+          - mariadb-server
+          - redis
+        state: absent
+
+    - name: deletings mysql
+      file:
+        path: /var/lib/mysql
+        state: absent
+
+    - name: Install Redis
```

Comment:
I would suggest installing all packages in one task; it doesn't sacrifice readability and runs fewer dnf transactions, which makes the execution much faster.